src/app/[locale]/guides/permission-sets/en.mdx

```diff
@@ -0,0 +1,195 @@
+import {Container} from "@/components/Container"
+import clearsky from "./clearsky.png"
+import skyblur from "./skyblur-oauth.png"
+import granular from "./skyblur-granular.png"
+import lexicon from "./skyblur-lexicon.png"
+
+export const metadata = {
+  title: 'Permission Set Guide',
+  description:
+    'A guide to requesting permissions and permission sets',
+}
+
+# Permission Sets
+
+## For app developers
+
+Control over AT Proto resources is described and granted using "permissions". A group of a permissions related to a specific Lexicon namespace (record types and API endpoints) can be bundled together as a "permission set". Both are used in the context of [OAuth](/guides/oauth) to grant client software access to account resources on a PDS: For example, the ability to write records of specific types to the user's public repository, or make authenticated API requests to remote services. Developers declare the permissions their app requires to function, and end users are shown the permissions when granting access to the app.
+
+In general, it is best to use existing permission sets for cleaner UI, but you can also request individual permissions and define your own permission sets as needed. Refer to the [for users](#for-users) section for how these requests appear in the OAuth flow.
+
+### Requesting permissions
```

Comment:
I think this section should mention OAuth more explicitly, and link to some high-level OAuth docs entrypoint for client devs